### PR TITLE
Decouple other modules from frontend:file

### PR DESCRIPTION
**Pull request by PawelLipski**
_Okrągły numer_ 🤓

### CODE REVIEW

File: frontend/actions/build.gradle.kts [hunk 1]
```diff
@@ -5,10 +5,9 @@ dependencies {
   implementation(project(":binding"))
   implementation(project(":backend:api"))
   implementation(project(":branchLayout:api"))
-  implementation(project(":frontend:ui:api"))
-  implementation(project(":frontend:file"))
```
Comment (PawelLipski) [inline frontend/actions/build.gradle.kts:9]: Wasn't even needed up to this point

File: frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMachetePanel.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public GitMachetePanel(Project project) {
       public void after(java.util.List<? extends VFileEvent> events) {
         for (val event : events) {
           if (event instanceof VFileContentChangeEvent) {
-            if (MacheteFileType.instance.equals(((VFileContentChangeEvent) event).getFile().getFileType())) {
+            if (((VFileContentChangeEvent) event).getFile().getFileType().getName().equals(FileTypeIds.NAME)) {
```
Comment (PawelLipski) [inline frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMachetePanel.java:58]: That was the only place where it was used

File: frontend/actions/build.gradle.kts (hunk 1 above)
Comment (PawelLipski) [inline frontend/actions/build.gradle.kts:9]: Wasn't even needed up to this point

Comment (PawelLipski) [inline frontend/actions/build.gradle.kts:9]: Wasn't even needed up to this point

**Review by MaciejG604 — APPROVED** [review]

**Pull request merged**